src/jobflow_remote/config/settings.py

```diff
@@ -30,6 +30,10 @@ class JobflowRemoteSettings(BaseSettings):
     cli_log_level: LogLevel = Field(
         LogLevel.WARN, description="The level set for logging in the CLI"
     )
+    cli_job_list_columns: list[str] = Field(
+        default_factory=list,
+        description="The list of columns to show in the `jf job list` command.",
```

Comment:
Actually I would like to avoid hard-coding those values. But importing them will give a circular import error. So either I need to move the map definition or it could point to the help page of the list command. What do you think?

Reply:
I think that referring to the help of `jf job list` for the list of available options should be enough.

Reply:
Great.